packages/uiweb/src/lib/context/spacesContext.ts

```diff
@@ -1,9 +1,19 @@
 import React, { createContext } from 'react';
 
+export interface ISpaceBannerData {
```

Comment:
instead of this, ig making a indexed object having spaceId as key and making it more generic would be a better way to cache data, 
maybe like this,
import SpaceDTO type from restapi

spacesInfo: { [key: string]: SpaceDTO }; // spaceId -> spaceDTO obj
setSpacesInfo: (spacesInfo: { [key: string]: SpaceDTO }) => void; // NOTE: set the appropriate type here